src/Navigation/Search/styled.js

```diff
@@ -0,0 +1,54 @@
+import styled from "styled-components";
+import {ReactComponent as searchIcon} from "../../assets/icon-search.svg";
+
+export const Wrapper = styled.div`
+    display: flex;
+    align-items: center;
+    
+    @media (max-width: ${({ theme }) => theme.breakpoints.tablet}px){    
+        margin: 0 auto;
+        display: grid;
+        grid-template-columns: 1fr;
+        max-width: 700px;
+        width: 100%;
+        padding: 30px;
+    };
+`;
```

Comment:
For screens above 992px, the search icon overlaps the placeholder

Reply:
i have an idea:
maybe we should style the Wrapper with `border-radius:25px;` and white background and then position search icon and input relatively to the Wrapper? 
Any way the search component should be wider not max: 400px :)